html/user/get_project_config.php

```diff
@@ -63,6 +65,14 @@ function show_platforms() {
     <web_rpc_url_base>".secure_url_base()."</web_rpc_url_base>
 ";
 
+if ( isset($git_commit) ) {
+    echo "<git_commit>$git_commit</git_commit>\n";
```

Comment:
Maybe have this a hyperlink to https://github.com/BOINC/boinc/commit/$git_commit ?